tasty-query/shared/src/test/scala/tastyquery/ReadTreeSuite.scala

```diff
@@ -51,13 +51,47 @@ class ReadTreeSuite extends RestrictedUnpicklingSuite {
   private object TermRefInternal:
     def unapply(tpe: TermRef): Some[(Type, AnyDesignator)] = Some((tpe.prefix, tpe.designatorInternal))
 
+  /** Extractors for `Type`s. */
+  private object ty:
```

Comment:
What does “ty” stand for?

Reply:
"type", but short, because we need it in many places in this file, and making it longer would clutter the actual tests.